### PR TITLE
Remove unused files

### PR DESCRIPTION
**Pull request by niranjanyardi**

### CODE REVIEW

**Review by y4vor — APPROVED** [review]

File: components/metrics/BUILD.gn [hunk 1]
```diff
@@ -20,8 +20,6 @@ static_library("metrics") {
     "data_use_tracker.h",
     "delegating_provider.cc",
     "delegating_provider.h",
-    "drive_metrics_provider.cc",
```
Comment (andrewsavage1) [inline components/metrics/BUILD.gn:23]: These should not be removed directly but in a use_cobalt_customizations block below

**Review by andrewsavage1 — CHANGES_REQUESTED** [review]

File: components/metrics/BUILD.gn (hunk 1 above)
Comment (andrewsavage1) [reply components/metrics/BUILD.gn:23]: (same with the other cases)

File: components/metrics/BUILD.gn [hunk 2]
```diff
@@ -20,8 +20,6 @@ static_library("metrics") {
     "data_use_tracker.h",
```
Comment (andrewsavage1) [inline components/metrics/BUILD.gn:20]: please revert your formatting to this file and the other, it's third party code. They can be added to the list of excluded directories in .pre-commit-config.yaml if they haven't already

File: components/metrics/BUILD.gn (hunk 1 above)
Comment (niranjanyardi) [reply components/metrics/BUILD.gn]: done

File: components/metrics/BUILD.gn (hunk 2 above)
Comment (niranjanyardi) [reply components/metrics/BUILD.gn]: yup, done. realized that as soon as i saw my cl on github. thanks

**Review by andrewsavage1 — APPROVED** [review]
Looks good! @joeltine you should probably take a quick look to make sure there's nothing you expect these methods to be used for currently

**Comment by niranjanyardi** [comment]
> Looks good! @joeltine you should probably take a quick look to make sure there's nothing you expect these methods to be used for currently

@joeltine  can you ptal

**Comment by joeltine** [comment]
As long as all the black box tests, unit tests, and builds are working, this should be fine. Can you confirm that's the case?

**Comment by andrewsavage1** [comment]
I didn't think of this at first, but a better title for this probably would've been "Remove files with unused symbols" to convey this isn't just a cleanup, but a necessity (just something to keep in mind if we do this again)

**Pull request merged**